### PR TITLE
Update bash aliases to work better with pulp 3/django

### PR DESCRIPTION
**Pull request by seandst**
fixes #2194
https://pulp.plan.io/issues/2194

### CODE REVIEW

File: ansible/roles/dev/files/bashrc [hunk 1]
```diff
@@ -116,20 +78,22 @@ pjournal() {
...
     pjournal takes optional journalctl args e.g. 'pjournal -r', runs pjournal -f by default"
 
+ptest() {
+    # based on the test running in the travis conf yaml
+    # travis should run the same tests devs do, but that gets a little tricky
+    # with Django, tracked in https://pulp.plan.io/issues/2266
+    # For now, this is a reasonable approximation until we figure out how to test pulp 3.
+    workon pulp
+    flake8 --config flake8.cfg app
+    python manage.py makemigrations pulp
```
Comment (asmacdo) [inline ansible/roles/dev/files/bashrc]: `python manage.py migrate` ?

File: ansible/roles/dev/files/bashrc [hunk 2]
```diff
@@ -53,46 +53,8 @@ preset() {
...
-    if [ -d /var/spool/squid ]; then
-        sudo systemctl stop squid
-        sudo bash -c "rm -rf /var/spool/squid/*"
-        sudo systemctl start squid
-    fi
-
-    sudo -u apache pulp-manage-db;
-    setup_crane_links;
-    pstart;
-    ppopulate;
```
Comment (asmacdo) [inline ansible/roles/dev/files/bashrc]: I like that cleaning the database no longer repopulates it.

File: ansible/roles/dev/files/bashrc [hunk 3]
```diff
@@ -53,46 +53,8 @@ preset() {
...
-        repo="${REPOS[idx-1]}"
-        if [ -d $repo ]; then
-            pushd $repo
-            sudo ./pulp-dev.py -U
-            popd
-        fi
-    done
-
-    # Destroy mongo DB and clean up pulp-related files
-    mongo pulp_database ~/drop_database.js;
```
Comment (asmacdo) [inline ansible/roles/dev/files/bashrc]: I think this is the only use of this file. Could you also remove the file and the ansible copy task?

File: ansible/roles/dev/files/bashrc (hunk 1 above)
Comment (seandst) [inline ansible/roles/dev/files/bashrc]: Nope. The tests make their own database, they just need the migrations made.

File: ansible/roles/dev/files/bashrc (hunk 3 above)
Comment (seandst) [inline ansible/roles/dev/files/bashrc]: Confirmed, I'm only seeing it in this file and ansible. Removing it now.

File: ansible/roles/dev/files/bashrc (hunk 1 above)
Comment (seandst) [inline ansible/roles/dev/files/bashrc]: I'm adding a comment to this effect before merging

**Pull request merged**